src/pages/People/People.css

```
@@ -0,0 +1,121 @@
+h1 {
+    text-align: center;
+}
+h4 {
+    color: #535353;
+}
+
+.main-content {
+    margin-top: 5.3rem;
+}
+
+.project-header {
+    margin-top: 10rem;
+}
+
+.page-description {
+    margin: 2rem 0;
+}
+
+.page-nav {
+    width: 70%;
+    margin: auto;
+    display: grid;
+    justify-content: center;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+}
+
+.nav-item {
+    border-left: 1px rgba(0, 0, 0, 0.2) solid;
+} 
+
+.hover-item {
+    border-radius: 5px;
+    margin: 0 .5rem;
+    transition: background-color 1s ease-in;
+    transition: color 0.1s ease-out;
+    padding: .5rem 0;
+    user-select: none;
+}
+
+.hover-item:hover {
+    cursor: pointer;
+    background-color: rgb(28, 66, 109);
+    color: white;
+}
+
+.selected-item {
+    background-color: rgb(28, 66, 109);
+    color: white;
+}
+
+.page-nav .nav-item:first-child {
+    border-left: none; /* Remove border from the first item */
+}
+
+.member-list {
+    width: full;
+    padding: 1rem 0;
+}
+
+.member {
+    width: 55%;
+    height: 10rem;
+    margin: auto;
+    margin-top: 1rem;
+    display: flex;
+    color: black;
+    border: 1px rgba(0, 0, 0, 0.05) solid; 
+    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
+    padding: 1rem 0;
+}
+
+.image-container {
+    width: 20%;
+}
+
+.image {
+    width: 60%;
+    background-color: black;
+    border-radius: 100px;
+}
+
+.info-container {
+    width: 80%;
+}
+
+.name {
+    color: black;
+}
+
+h3 {
```

Since this `h3` element wasn't given a name in the .tsx file, you can refer to it in a specific manner like so: 

```suggestion
.position h3 {
```

In which position is its parent element's `className`! :)